File: core/src/main/scala/pl/edu/agh/multiscalemodelling/processsimulation/Simulation.java
package pl.edu.agh.multiscalemodelling.processsimulation;

import pl.edu.agh.multiscalemodelling.engine.Application;

public class Simulation extends Application {

    @Override
    public void create() {

        super.create();
        setScreen(new SimulationScreen(this));

    }

    @Override
    public void render() {
        super.render();
    }

    @Override
    public void dispose() {

        super.dispose();

    }

}

File: core/src/main/scala/pl/edu/agh/multiscalemodelling/processsimulation/Simulation.scala
package pl.edu.agh.multiscalemodelling.processsimulation

import pl.edu.agh.multiscalemodelling.engine.Application

class Simulation extends Application {
  override def create(): Unit = {
    super.create()
    setScreen(new SimulationScreen(this))
  }

  override def render(): Unit = super.render()

  override def dispose(): Unit = super.dispose()
}